tests/foreman/cli/test_satellitesync.py

```diff
@@ -149,15 +149,15 @@ def test_positive_export_complete_version_custom_repo(
         assert len(cv['versions']) == 1
         cvv = cv['versions'][0]
         # Verify export directory is empty
-        assert validate_filepath(default_sat, module_org) == ''
+        assert validate_filepath(target_sat) == ''
```

Comment:
Change to the function call here, and `org` is a required param in the func signature.  @rmynar please review.

Reply:
reverted since this usage is based on the newer `validated_filepath`.
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/cli/test_satellitesync.py#L398